src/schema/submission.yaml

```diff
@@ -139,57 +196,23 @@ classes:
         description: >-
           The type of this Sequencing Experiment.
         required: false
-      sequencing_protocol:
-        description: >-
-          The Sequencing Protocol associated with this Sequencing Experiment.
-        required: true
-        inlined: false
-      library_preparation_protocol:
-        description: >-
-          The Library Preparation Protocol associated with this Sequencing Experiment.
+      ega_accession:
+        required: false
+      sample:
+        description: The alias of one or more Samples that are associated with this Experiment.
         required: true
         inlined: false
 
-  Condition:
+  ExperimentalMethod:
     mixins:
       - IdentifiedByAliasMixin
       - AttributeMixin
     description: >-
-      A Condition specifies which special characteristics and treatments apply to a Sample (e.g., whether the Sample comes from a disease or a healthy tissue).
+      The Experimental Method captures the technical parameters that were used to produce output from the Sample.
```

Comment:
Produce output sounds a bit rough. Maybe something along the lines of "..captures technical metadata describing the data acquisition for a sample"?